.gitlab-ci.yml

```
@@ -52,22 +52,29 @@ test:build_docs:
     - make html
     - stat _build/html/index.html
 
-test:py36:
+test:py37:
```

It looks like the test job names are not aligned with the image being used (off-by-one too low).
```suggestion
test:py38:
```